.github/dependabot.yml

```diff
@@ -4,6 +4,8 @@ updates:
     directory: "/"
     schedule:
       interval: daily
+    versioning-strategy: increase
```

Comment:
actually both of these keys are not required according to the [docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file). Just curious why adding this 2 fixes the issue?

Reply:
im not exactly sure but i was wondering why the bot isn't triggering and seems like someone else had the same issue and was fixed by adding `versioning-strategy` (https://github.com/dependabot/dependabot-core/issues/5226) so I thought I'd give it a try

the other one `open-pull-requests-limit` had nothing to do with the issue but a good to have, to increase the limit from the default 5 to 10. Once the limit is hit, it stopped opening a new PR and from experience it's very easy to hit the default 5 limit

Reply:
alright